unix/lib/time.mli

```diff
@@ -18,6 +18,6 @@ type +'a io = 'a Lwt.t
 
 (** Timeout operations. *)
 
-val sleep : float -> unit Lwt.t
```

Comment:
I just wrote up a [durations](https://github.com/hannesm/durations) library for easy conversion from your favourite time representation (microseconds, milliseconds, seconds, minutes, hours, years as int), seconds as floating point number) to nanoseconds (and the inverse).  A uint64 representing nanoseconds allows for a range up to 115 years, that should be sufficient for sleep durations.

IMHO `OCaml uses floats` is a mistake (and the queued changes to PCLOCK/MCLOCK inside of mirage-types will get rid of this mistake inside of mirage).